index.html

```diff
@@ -5,36 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Oleo+Script+Swash+Caps&family=Open+Sans:wght@400;700&family=Poppins:ital,wght@0,200;0,300;0,500;0,600;0,700;0,800;1,200&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <title>Josue-Portofolio</title>
 </head>
 <body>
     <header class="header">
-        <div class="header-container">
-            <nav class="nav-item1"><a class="Mylogo" href="">My Logo</a></nav>
-            <nav class="nav-item2"></nav>
-            <nav class="nav-item3">
-                <img class="toggle" src="images/toggle.png" alt="">
-            </nav>
-            <div id="myNav" class="overlay">
-                <a class="closebtn"> &times; </a>
-                <div class="overlay-content">
-                    <a class="pop-nav" href="#portfolio">Portfolio</a>
-                    <a class="pop-nav" href="#about">About</a>
-                    <a class="pop-nav" href="#contact">Contact</a>
-                </div>
-            </div>
-        </div>
-        <nav class="web-nav">
-            <div class="logo">
-                <a class="Mylogo" href="">My Logo</a>
-            </div>
+        <nav >
+          
+            <div class="logo"><a class="Mylogo" href=""> MyLogo </a>
+             
             <ul>
-                <li><a href="#portfolio">Portfolio</a></li>
-                <li><a href="#about">About</a></li>
-                <li><a href="#contact">Contact</a></li>
-            </ul>
-        </nav>
+              <li>
+                  <a href="#">Portfolio</a>
+              </li>
+  
+              <li>
+                  <a href="#">About</a>
+              </li>
+  
+              <li>
+                  <a href="#">Contact</a>
+              </li>
+  
```

Comment:
- [ ] Good job working on the functionality of the menu but  please make sure that when any of the mobile menu options, a correct part of the page is displayed, for instance, when I click on the `about` option, the page about section should be displayed :blush: 
![Screenshot from 2023-01-03 16-21-33](https://user-images.githubusercontent.com/92303891/210397788-30e9136f-bc0f-4114-9b4c-5c5c2075495a.png)